archive/imo/imo1969_q1.lean

```diff
@@ -0,0 +1,81 @@
+import tactic
+import tactic.basic
```

Comment:
I think `import tactic` covers all of these; you don't need the other imports.

Reply:
There was a PR recently that replaced all `import tactic` with more specific `tactic.*` imports.

Reply:
I replaced this set of imports with only the three that are necessary, I think that will be most consistent with the surrounding style.